presentation/src/main/java/com/architect/coders/mu8/ui/common/Scope.kt

```diff
@@ -0,0 +1,26 @@
+package com.architect.coders.mu8.ui.common
+
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.Job
+import kotlinx.coroutines.SupervisorJob
+import kotlin.coroutines.CoroutineContext
+
+interface Scope : CoroutineScope {
```

Comment:
Please check my PR, I'm getting rid of this `Scope` in order to create a `ScopedViewModel` to use.

Reply:
done :D